src/inputs/file-input.jsx

```diff
@@ -63,6 +63,9 @@ class FileInput extends React.Component {
   updateText = e => {
     e.persist()
     const fileElem = document.getElementById(this.props.id)
+
+    if (!fileElem.files.length) return
+
     let fileNames = [...fileElem.files].map(file => file.name)
     console.log('this.props.multiple', this.props.multiple)
```

Comment:
No more console logs plz

Reply:
![car-attacks-logs-Hw5wyPEsNgxc4](https://media2.giphy.com/media/Hw5wyPEsNgxc4/giphy.gif?cid=e1bb72ff5aa693fe5a5650485151b629) ... But this is not my log